TESTING.md

```diff
@@ -0,0 +1,59 @@
+# Testing 'oneview' cookbook
+The 'oneview' cookbook provides already unit testing strategy, besides having linting tools.
+This document will cover on how to execute and implement these tests.
+
+## Executing tests
+The tests can be executed by `rake` tasks, `guard` watches, and by their original tools, like `rspec`, `chefspec`, `rubocop` and `foodcritic`.
+
+### Rake
+All the test strategies and checks can be executed by the rake command.
+Please use `rake -T` to see a full list of commands.
+
+### Guard
+Guard will watch the changes and execute unit tests and style checks. Just activate it to have real time test execution on every change.
+
+### Unit tests
+All unit tests are inside spec folder. You can execute them manually by using the RSpec and specifying the test files, like `rspec spec/path/to/my/tests`.
```

Comment:
All unit tests are inside **the** spec folder. You can execute them manually by using ~the~ RSpec and specifying the test files, like `rspec spec/path/to/my/tests`.